C++: mention DD_APM_RECEIVER_SOCKET earlier in the setup doc (#13256)
DataDog/documentation: content/en/tracing/setup_overview/setup/cpp.md

## Configure the Datadog Agent for APM

Install and configure the Datadog Agent to receive traces from your instrumented application.

> By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic at `localhost:8126`. The Agent may be configured to listen on a unix domain socket instead by setting the environment variable `DD_APM_RECEIVER_SOCKET` to the desired socket file path.

Reviewer: ```suggestion
By default, APM is enabled in the `datadog.yaml` file under `apm_config` as `enabled: true`. The Datadog Agent listens for trace traffic at `localhost:8126`. The Agent may be configured to listen on a Unix domain socket instead by setting the environment variable `DD_APM_RECEIVER_SOCKET` to the desired socket file path.
```
